pages/code-view/with-line-wrapping.page.tsx

```diff
@@ -0,0 +1,45 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { SpaceBetween } from "@cloudscape-design/components";
+
+import { CodeView } from "../../lib/components";
+import cppHighlight from "../../lib/components/code-view/highlight/cpp";
+import { ScreenshotArea } from "../screenshot-area";
+
+export default function CodeViewPage() {
+  return (
+    <ScreenshotArea>
+      <h1>Code View</h1>
+      <SpaceBetween direction="vertical" size="l">
+        No wrapping, no line numbers
```

Comment:
Could you please update this page and also `with-syntax-highlighting.page.tsx` with wrapping text content inside SpaceBetween in some container, e.g. a Box? That is to satisfy the guidelines: https://cloudscape.aws.dev/components/space-between/?tabId=api

Currently, there are React warnings in these pages.

Reply:
ah yeah good point, fixed in new commit